protocol/x/clob/ante/clob.go

```diff
@@ -130,6 +159,17 @@ func (cd ClobDecorator) AnteHandle(
 			return next(ctx, tx, simulate)
 		}
 
+		// HOTFIX: Ignore any short-term clob messages in a transaction with a timeout height.
```

Comment:
For my understanding? Why do we need hotfix for `MsgBatchCancel` and `MsgCancel`? They will never be attached in the operations queue, so do not go through the AnteHandler during `PrepareProposal`.

Reply:
True, that can be removed.